src/features/list-view/ListView.js

```diff
@@ -28,21 +33,34 @@ type CellRendererArgs = {|
 
 class ListView extends React.PureComponent<Props> {
     cellRenderer = ({ columnIndex, key, rowIndex, style }: CellRendererArgs) => {
-        const { getGridCell, getGridHeader } = this.props;
+        const { getGridCell, getGridHeader, getGridHeaderSort, onSortChange } = this.props;
 
         if (rowIndex === 0) {
             const displayName = getGridHeader(columnIndex);
+            const sortDirection = getGridHeaderSort && getGridHeaderSort(columnIndex);
```

Comment:
Could default `getGridHeaderSort` to `noop` when extracting it, the simplify this to 

```
const sortDirection = getGridHeaderSort(columnIndex);
```